packages/dds/tree/src/simple-tree/arrayNode.ts

```diff
@@ -760,19 +728,13 @@ export function arraySchema<
 		public get [type](): TName {
 			return identifier;
 		}
+
+		protected get simpleSchema(): T {
+			return info;
+		}
 	}
 
-	// Setup array functionality
-	Object.defineProperties(schema.prototype, arrayNodePrototypeProperties);
-
-	return schema as unknown as TreeNodeSchemaClass<
```

Comment:
this unsafe conversion is no longer needed since the implementation and APi are aligned type wise. THats a big safty improvement.

Reply:
Woohoo!